controllers/managedocs_controller_test.go

```diff
@@ -475,6 +477,38 @@ var _ = Describe("ManagedOCS controller", func() {
 	}
 
 	Context("reconcile()", func() {
```

Comment:
- pls add more description here
- `reconcile converged mode` would be good enough